lib/css/CssParser.js

```diff
@@ -560,6 +564,9 @@ class CssParser extends Parser {
 							modeData = "local";
 							const dep = new ConstDependency("", [start, end]);
 							module.addPresentationalDependency(dep);
+						} else if ([":not", ":is"].includes(name)) {
```

Comment:
Why noyl `not` and `is`, there are many of them...

- old `matches`
- `is`, `where`, `has`
- `current`, `past`, `future`
- `-moz-any"` and `-webkit-any`

Reply:
Okay will add as well

Reply:
Let's add a test case to make sure everything works fine, should be easy :smile:

Reply:
done